src/vse_sync_pp/analyzers/analyzer.py

```diff
@@ -265,3 +270,193 @@ def explain(self, data):
             'duration': data.iloc[-1].timestamp - data.iloc[0].timestamp,
             'terror': self._statistics(data.terror, 'ns'),
         }
+
+
+class TimeIntervalErrorAnalyzerBase(Analyzer):
+    """Analyze Time Interval Error (also referred to as Wander).
+
+    Derived classes calculate specific Time Interval Error metric focused on measuring
+    the change of Time Error.
+    """
+    locked = frozenset()
+
+    def __init__(self, config):
+        super().__init__(config)
+        # samples in the initial transient period are ignored
+        self._transient = config.parameter('transient-period/s')
+        # minimum test duration for a valid test
+        self._duration = config.parameter('min-test-duration/s')
+        # limit initial tau observation windows from 1 to 9k taus
+        taus_below_tenkey = np.array([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 20, 30, 40, 50, 60, 70, 80, 90,
```

Comment:
I think it's ok to have numbers in the variable name, as long as it's not starting with it.
WDYT about this?
```suggestion
        taus_below_9k = np.array([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 20, 30, 40, 50, 60, 70, 80, 90,
```